<launch>
  <arg name="debug" default="false"/>
  <arg name="hz" default="100"/>
  <arg name="max_velocity" default="100"/>
  <arg name="scale" default="3"/>
  <arg name="min_distance" default="10"/>
  <arg name="upper_obstacle_filter" default="7"/>
  <arg name="lower_obstacle_filter" default="1"/>
  <arg name="threthold" default="5"/>
  <arg name="arrival_time" default="5"/>
  <arg name="wheel_base" default = "1"/>

  <node pkg="scc" type="scc" name="scc" output="screen">
    <param name="debug" value="$(arg debug)"/>
    <param name="hz" value="$(arg hz)"/>
    <param name="max_velocity" value="$(arg max_velocity)"/>
    <param name="scale" value="$(arg scale)"/>

    <param name="min_distance" value='$(arg min_distance)'/>
    <param name="upper_obstacle_filter" value="$(arg upper_obstacle_filter)"/>
    <param name="lower_obstacle_filter" value="$(arg lower_obstacle_filter)"/>

    <param name="threthold" value="$(arg threthold)"/>
    <param name="arrival_time" value="$(arg arrival_time)"/>
    <param name="wheel_base" value="$(arg wheel_base)"/>
  </node>
</launch>
